srcpkgs/incus/template

```diff
@@ -0,0 +1,68 @@
+# Template file for 'incus'
+pkgname=incus
+version=0.4.0
+revision=1
+build_style=go
+go_import_path=github.com/lxc/incus
+go_build_tags=libsqlite3
+go_package="${go_import_path}/cmd/incus
+ ${go_import_path}/cmd/lxc-to-incus
+ ${go_import_path}/cmd/incus-benchmark
+ ${go_import_path}/cmd/incusd
+ ${go_import_path}/cmd/incus-user"
+# temporarily don't build fluidshift, as it conflicts with lxd, lxd-lts
+# later on it should be part of incus-tools
+hostmakedepends="pkg-config"
+makedepends="lxc-devel acl-devel cowsql-devel raft-devel
+ libcap-devel libuv-devel sqlite-devel eudev-libudev-devel"
+depends="lxc acl acl-progs rsync squashfs-tools xz dnsmasq iptables attr-progs"
+short_desc="Powerful system container and virtual machine manager"
+maintainer="dkwo <npiazza@disroot.org>"
+license="Apache-2.0"
+homepage="https://linuxcontainers.org/incus"
+distfiles="https://github.com/lxc/incus/archive/refs/tags/v${version}.tar.gz"
+checksum=1195d8aecaf838806b88580dfc8b48302dcbb4612addcb6d6bfa480f14d97a0d
+system_groups="incus-admin incus"
```

Comment:
why isn't this following the [established convention](https://github.com/void-linux/void-packages/blob/master/Manual.md#creating-system-accountsgroups-at-runtime) for system groups?